docs/commands.md

```diff
@@ -3214,10 +3214,14 @@ hal config provider kubernetes account add ACCOUNT [parameters]
 When no context is configured for an account the 'current-context' in your kubeconfig is assumed.
  * `--deployment`: If supplied, use this Halyard deployment. This will _not_ create a new deployment.
  * `--docker-registries`: (*Default*: `[]`) A list of the Spinnaker docker registry account names this Spinnaker account can use as image sources. These docker registry accounts must be registered in your halconfig before you can add them here.
+ * `--kinds`: (*Default*: `[]`) (V2 Only) A list of kinds this Spinnaker account can deploy to and will cache.
+When no kinds are configured, this defaults to 'all kinds described here https://spinnaker.io/reference/providers/kubernetes-v2'.
  * `--kubeconfig-file`: The path to your kubeconfig file. By default, it will be under the Spinnaker user's home directory in the typical .kube/config location.
  * `--namespaces`: (*Default*: `[]`) A list of namespaces this Spinnaker account can deploy to and will cache.
 When no namespaces are configured, this defaults to 'all namespaces'.
  * `--no-validate`: (*Default*: `false`) Skip validation.
+ * `--omit-kinds`: (*Default*: `[]`) (V2 Only) A list of kinds this Spinnaker account cannot deploy to or cache.
+This can only be set when no --kinds are provided.
```

Comment:
"When --kinds is empty or not set", perhaps? It seems "[]" is a valid value for --kinds when --omit-kinds is non-empty.